spec/factories/action_factory.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
```

Comment:
I would love to see this when this property was introduced, but not a deal breaker.